plasmapy/formulary/collisions.py

```diff
@@ -2033,3 +2036,63 @@ def coupling_parameter(
         )
 
     return coulomb_energy / kinetic_energy
+
+
+class RelaxationRates:
+    @validate_quantities(
+        n_b={"can_be_negative": False},
+        T_b={"can_be_negative": False, "equivalencies": u.temperature_energy()},
+    )
+    def __init__(
+        self,
+        species: (particles.Particle, particles.Particle),
```

Comment:
Suggest having 'test_particle' and 'background_particle' or something as separate arguments for clarity, since the distinction between the a and b species is subtle but very important.